pvlib/tests/test_conftest.py

```diff
@@ -19,3 +20,24 @@ def test_fail_on_pvlib_version_pass():
 @fail_on_pvlib_version('100000.0')
 def test_fail_on_pvlib_version_fail_in_test():
     raise Exception
+
+
+# set up to test passing arguments to conftest.fail_on_pvlib_version
+@pytest.fixture(scope='function')
+def some_data():
+    return "some data"
+
+
+def alt_func(*args):
+    print(args[0])
+    return
```

Comment:
instead of print you could return the args and then assert that they were returned in the calling test.